components/chef-automate-collect/go.mod.bak

```diff
@@ -0,0 +1,10 @@
+module github.com/chef/chef-workstation/components/chef-automate-collect
```

Comment:
Lets leave this file out. We don't need to revert back to this for any reason and I don't see anything online about a `go.mod.bak`.

Reply:
Deleted `go.mod.bak`